react/features/base/react/components/native/SlidingView.js

```diff
@@ -183,7 +187,7 @@ export default class SlidingView extends PureComponent<Props, State> {
             ...this.props.style,
             ...styles.sliderViewContent
         };
-        const { positionOffset } = this.state;
+        const positionOffset = Math.max(this.state.positionOffset, 0);
```

Comment:
do we want to do this in the end?

Reply:
I think it's not necessary, but makes sense. What do you think?

Reply:
Nevermind, I removed it, don't want it to cause any troubles that we just don't foresee yet